.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "\U0001F914 Question"
+about: Usage question that isn't answered in docs or discussion
+
+---
+
+## Question
+
+Before asking a question, make sure you have:
+
+- Searched existing Stack Overflow questions.
+- Googled your question.
```

Comment:
what keywords should be used?

Reply:
Yes, I admit that we don't have many users right now. So the search via Google and StackOverflow may not suitable right now. I can remove it if you think it's unnecessary and add it back when needed.

Reply:
```suggestion
- Reviewed relevant Kubernetes information: Google your error messages and look at K8s docs.
```

Reply:
Yes, we should decide if we want to use Stack Overflow or github issues.